experimental/iterators/lib/Conversion/IteratorsToLLVM/IteratorAnalysis.cpp

```diff
@@ -93,11 +101,11 @@ mlir::iterators::IteratorAnalysis::IteratorAnalysis(Operation *rootOp)
           auto stateType = computeStateType(op);
           setIteratorInfo(op, IteratorInfo(op, nameAssigner, stateType));
         })
-        /// The state of ReduceOp only consists of the state of its upstream
-        /// iterator, i.e., the state of the iterator that produces its input
-        /// stream.
-        // TODO: ReduceOp verifier that op.input does not come from a bbArg.
-        .Case<ReduceOp>([&](auto op) {
+        /// The respective state of FilterOp and ReduceOp only consist of the
```

Comment:
I would make this comment generic in the op name now, to avoid updating each and every time a new op gets added with this property.

Reply:
The commit *"Make signature and usage of computeStateType independent of Op type."* in #554 fixes this, which you approved in the meantime, so I'll consider this solved.